types/lodash/common/common.d.ts

```diff
@@ -9,6 +9,7 @@ declare module "../index" {
     type ImpChain<T> =
         T extends { __trapAny: any } ? Collection<any> & Function<any> & Object<any> & Primitive<any> & String :
         T extends null | undefined ? never :
+        T extends string ? String<T> :
         T extends string | null | undefined ? String :
```

Comment:
Conditional types distribute over naked generic types in on the left-hand side of `extends`. Since `null | undefined` are handled on line 11, and now that `string` is handled on line 12, this branch will never be hit. The `null | undefined` in the `extends` type on line 15 is also unnecessary.